core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2020
```

Comment:
Copyright needs to mention the copyright holders. If you are not comfortable with using your name here, consider adding "Jenkins contributors"

Reply:
Thank you, I have added my name :)